### PR TITLE
[12.0][MIG] sale_commission_formula

### PR DESCRIPTION
**Pull request by novawish**

### CODE REVIEW

File: sale_commission_formula/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+==========================
```
Comment (pedrobaeza) [inline sale_commission_formula/readme/DESCRIPTION.rst]: This should go out

File: sale_commission_formula/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Usage
```
Comment (pedrobaeza) [inline sale_commission_formula/readme/USAGE.rst]: This should go out

File: sale_commission_formula/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+=====
+
+To use this module, you need to:
+
+* Go to Sales, Commission Types and create a commission with type formula
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (pedrobaeza) [inline sale_commission_formula/readme/USAGE.rst]: Remove this part. It's automatically generated.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Is it possible for you to pre-generate the README?

**Comment by pedrobaeza** [comment]
Please fix also the demo data adjusting UoM to the new module.

File: sale_commission_formula/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# © 2016 Davide Corio - Abstract
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+{
+    'name': 'Sale Commission Formula',
+    'version': '12.0.1.0.0',
+    'category': 'Sale',
+    'license': 'AGPL-3',
+    'summary': 'Sale commissions computed by formulas',
+    'author': "Abstract,Odoo Community Association (OCA)",
+    'website': 'http://www.abstract.it',
```
Comment (tbaden) [inline sale_commission_formula/__manifest__.py]: please change website to: https://github.com/OCA/commission

File: sale_commission_formula/views/sale_commission_view.xml [hunk 5]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+      <template id="assets_backend" name="sale_commission_formula_assets_backend" inherit_id="web.assets_backend">
+          <xpath expr="." position="inside">
+              <link rel="stylesheet" href="/sale_commission_formula/static/src/css/sale_commission_formula.css"/>
+          </xpath>
+      </template>
+
+        <record model="ir.ui.view" id="sale_commission_form_formula">
```
Comment (tbaden) [inline sale_commission_formula/views/sale_commission_view.xml]: optional change:
id before model

use original id (sale_commission_form)

> prefer naming the inherited view after the original over using a name which follows the guidelines. This eases looking up the original view and other inheritance if they all have the same name.

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for  the work 👍 
I did a code review: just a small change and an optional if you like.

**Comment by novawish** [comment]
@pedrobaeza @tbaden Could you please review again?

**Review by tbaden — APPROVED** [review]
LGTM

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**